sdk/eventhub/event-hubs/README.md

```diff
@@ -1,259 +1,198 @@
 
-- Getting debug logs from the Event Hub SDK
 ```bash
-export DEBUG=azure*
-```
-- Getting debug logs from the Event Hub SDK and the protocol level library.
-```bash
-export DEBUG=azure*,rhea*
-```
-- If you are **not interested in viewing the message transformation** (which consumes lot of console/disk space) then you can set the `DEBUG` environment variable as follows:
-```bash
-export DEBUG=azure*,rhea*,-rhea:raw,-rhea:message,-azure:amqp-common:datatransformer
-```
-- If you are interested only in **errors**, then you can set the `DEBUG` environment variable as follows:
-```bash
-export DEBUG=azure:event-hubs:error,azure-amqp-common:error,rhea-promise:error,rhea:events,rhea:frames,rhea:io,rhea:flow
+npm install @types/node
 ```
 
-#### Logging to a file
-- Set the `DEBUG` environment variable as shown above and then run your test script as follows:
-  - Logging statements from your test script go to `out.log` and logging statements from the sdk go to `debug.log`.
-    ```bash
-    node your-test-script.js > out.log 2>debug.log
-    ```
-  - Logging statements from your test script and the sdk go to the same file `out.log` by redirecting stderr to stdout (&1), and then redirect stdout to a file:
-    ```bash
-    node your-test-script.js >out.log 2>&1
-    ```
-  - Logging statements from your test script and the sdk go to the same file `out.log`.
-    ```bash
-      node your-test-script.js &> out.log
-    ```
+You also need to enable `compilerOptions.allowSyntheticDefaultImports` in your tsconfig.json. Note that if you have enabled `compilerOptions.esModuleInterop`, `allowSyntheticDefaultImports` is enabled by default. See [TypeScript's compiler options handbook](https://www.typescriptlang.org/docs/handbook/compiler-options.html) for more information.
+
+### Authenticate the client
+
+Interaction with Event Hubs start with an instance of the [EventHubClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest) class. You can instantiate
+this class using one of the 4 static methods on it
 
-## Examples ##
+- [createFromConnectionString](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest#createfromconnectionstring-string--ClientOptions-)
+  - This method takes the connection string and entity name to your Event Hub instance. You can get the connection string
+    from the Azure portal
+- [createFromIotHubConnectionString](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest#createfromiothubconnectionstring-string--clientoptions-)
+  - This method takes an IotHub Connection string. You can get the connection string from the Azure portal. This is useful for receiving telemetry data
+    of IotHub from the linked EventHub. Most likely the associated connection string will not have send claims. Hence getting HubRuntimeInfo or PartitionRuntimeInfo
```

Comment:
Can you point me to some documentation from where we got this?

Reply:
I got this from here: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/README.md#example-6---create-an-eventhubclient-from-an-iothub-connection-string

Also, I tested this - my iot connection string didn't have the send claims so I got this error:
 `Error occurred:  { UnauthorizedError: Unauthorized access. 'Send' claim(s) are required to perform this operation. }`

Similar issue: https://stackoverflow.com/questions/49806179/how-do-i-add-send-permission-to-my-iot-hub-connection-string

Reply:
Looks like this is a very specific use case than creating simple EventHubsClient. Lets remove it from this section and add it to the samples section

Reply:
Lets keep the old sample in our new samples section as is, create a new issue to dig into this more
Put that new issue under this epic: #1999